### PR TITLE
S05 fix select stats

### PR DESCRIPTION
**Pull request by cyrius83600**

### CODE REVIEW

**Review by OCR-VH-DevWeb — APPROVED** [review]
approved

File: src/app/app.routes.ts [hunk 1]
```diff
@@ -59,7 +59,7 @@ export const routes: Routes = [
   },
 
   {
-    path: 'stats',
+    path: 'creaPersonnage/stats',
```
Comment (Benja-san) [inline src/app/app.routes.ts:62]: use french complete words for prettier urls

File: src/app/pages/classes-page/classes-page.component.html [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+              <div class="statScore">{{stat.score}}</div>
+            </div>
+          </li>
+        }
+        </ul>
+        
+      </div>
+
+      <div class="selectClass">
+          <h1 class="title">Sélection des classes</h1>
```
Comment (Benja-san) [inline src/app/pages/classes-page/classes-page.component.html:22]: only one H1 per page

File: src/app/pages/classes-page/classes-page.component.html [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+            <div class="flexClasses">
+              <button class="content" class="buttonLeft" (click)="decrementCount()"><div class="circleClasses"><</div></button>
+                @for(filters of filterClasses; track filters)
```
Comment (Benja-san) [inline src/app/pages/classes-page/classes-page.component.html:25]: wrap in ul and use li

File: src/app/pages/classes-page/classes-page.component.html [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+                {
+                  <div class="flexPic" style="width:40%">
+                    <button (click)="changePic(filters.name)"><img [ngStyle] = "{'border-color': selectedClass.name === filters.name ? '#DDB375' : 'black'}"   style="height:50px;width:50px;border-radius:50px;border-style:solid;" src="{{filters.url}}"/></button>
+                    <div>{{filters.name}}</div>
+                  </div>
+                }
+                <button class="content" class="buttonRight" (click)="incrementCount()"><div class="circleClasses">></div></button>
+            </div>
+            
+            @for(description of selectedClass.content;track description){
```
Comment (Benja-san) [inline src/app/pages/classes-page/classes-page.component.html:35]: same here

**Review by Benja-san — CHANGES_REQUESTED** [review]

File: src/app/pages/classes-page/classes-page.component.html (hunk 2 above)
Comment (cyrius83600) [reply src/app/pages/classes-page/classes-page.component.html]: This file must not appear in that PR but i take into account your requests

File: src/app/pages/classes-page/classes-page.component.html (hunk 3 above)
Comment (cyrius83600) [reply src/app/pages/classes-page/classes-page.component.html]: Idem

File: src/app/pages/classes-page/classes-page.component.html (hunk 4 above)
Comment (cyrius83600) [reply src/app/pages/classes-page/classes-page.component.html]: Idem

File: src/app/pages/classes-page/classes-page.component.ts [hunk 5]
```diff
@@ -0,0 +1,113 @@
...
+  points!:number;
+
+  apiService = inject(ApiDataService);
+
+  changePic(name:String)
+  {
+    this.selectedClass = this.classes.filter(p => p.name === name)[0];
+  }
+
+  getIndexFromSelected():number
```
[the remark this answers is not in the document]
Comment (cyrius83600) [reply src/app/pages/classes-page/classes-page.component.ts]: Idem

[the remark this answers is not in the document]
Comment (cyrius83600) [reply src/app/pages/classes-page/classes-page.component.ts]: Classes and filterClasses are not the same.

**Review by Benja-san — APPROVED** [review]

**Pull request merged**